lib/src/rewrite.rs

```diff
@@ -190,6 +190,24 @@ impl<'repo> CommitRewriter<'repo> {
         }
     }
 
+    /// Update the intended new parents by replacing any occurrence of
+    /// `old_parent` by `unrewritten_parents`.
+    pub fn replace_rewritten_parent<'a>(
```

Comment:
nit: are we going to add non-test callers of this function?

Reply:
I've removed this function since it's only used in the tests.